src/connection_transfer/connection_transfer.go

```diff
@@ -36,22 +41,21 @@ func (connectionTransfer *ConnectionTransfer) Update(name string, up bool) {
 		IsConnected: up,
 	}
 
-	message, err := json.Marshal(mapToArray(connectionTransfer.boardStatus))
-	if err != nil {
-		log.Fatalf("connection transfer: Update: %s\n", err)
-	}
-
-	for id, socket := range connectionTransfer.sockets {
-		if err := socket.WriteMessage(websocket.TextMessage, message); err != nil {
-			socket.Close()
-			delete(connectionTransfer.sockets, id)
-		}
-	}
+	connectionTransfer.send()
 }
 
-func (ConnectionTransfer *ConnectionTransfer) Close() {
-	for _, socket := range ConnectionTransfer.sockets {
-		socket.Close()
+func (connectionTransfer *ConnectionTransfer) send() {
+	msg, err := json.Marshal(mapToArray(connectionTransfer.boardStatus))
+	if err != nil {
+		log.Printf("connectionTransfer: send: %s\n", err)
+		return
+	}
+	connectionTransfer.channel <- ws_models.MessageTarget{
```

Comment:
Create function that returns a `ws_models.MessageTarget` and use it here, in data transfer, etc.